### PR TITLE
Remove jenkins update center component

### PR DESCRIPTION
**Pull request by LinuxSuRen**
Don't need Jenkins update-center anymore. We can package all plugins with Jenkins core into a docker image. I started a new project which can archive that.
So once I confirm it's ok. Please fork [this project](https://github.com/LinuxSuRen/ks-jenkins/) into [kubesphere org](https://github.com/kubesphere/ks-installer). After that, I will remove my project.

These changes are dependent on the following upstream PRs:

* https://github.com/jenkins-zh/jenkins-cli/pull/458 was merged

## References
* https://github.com/kubesphere/ks-jenkins/
* https://github.com/jenkinsci/custom-war-packager

### CODE REVIEW

File: roles/ks-devops/jenkins/files/jenkins/jenkins/templates/config.yaml [hunk 1]
```diff
@@ -26,7 +26,7 @@ data:
       <viewsTabBar class="hudson.views.DefaultViewsTabBar"/>
       <myViewsTabBar class="hudson.views.DefaultMyViewsTabBar"/>
       <clouds>
-        <org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud plugin="kubernetes@{{ template "jenkins.kubernetes-version" . }}">
+        <org.csanchez.jenkins.plugins.kubernetes.KubernetesCloud plugin="kubernetes@1.19.2">
```
Comment (LinuxSuRen) [inline roles/ks-devops/jenkins/files/jenkins/jenkins/templates/config.yaml:29]: Don't have a perfect way to replace the hard coding way.

**Comment by LinuxSuRen** [comment]
I test against this feature via docker image `surenpi/test-ks-install:2`

**Comment by LinuxSuRen** [comment]
/assign shaowenchen

Hi @shaowenchen, please help me to review this PR.

**Comment by shaowenchen** [comment]
/LGTM It needs to be tested for installation.

**Pull request merged**